documentation/how-to/prepare-for-production.md

```diff
@@ -0,0 +1,193 @@
+
+# Prepare for Production
+
+## Performance Optimisation
+
+### Enable Asynchronous Querying
+To enable asynchronous querying, you need to deploy Charmed Superset Workers. These workers handle long-running queries asynchronously, allowing the UI to remain responsive. You can deploy additional Superset workers to offload and process background queries. Deploy Charmed Superset workers to the same model as the server with:
+
+```
+# Deploy Charmed Superset with worker functionality
```

Comment:
We avoid adding code comments. We use text to explain those if needed. This is also part of our style guide I linked above. This applies to all cases.
```suggestion

```

Reply:
I would break down the paragraph above into two if that makes sense as follows.
 First one:
To enable asynchronous querying, you need to deploy Charmed Superset Workers. These workers handle long-running queries asynchronously, allowing the User Interface (UI) to remain responsive. You can do so as follows:
``` juju deploy superset-k8s --config charm-function=worker superset-k8s-worker ```

Second one: 
For example, deploy Charmed Superset workers to the same model as the server:
``` 
juju relate superset-k8s-worker postgresql-k8s
juju relate superset-k8s-worker redis-k8s 
```